src/infection-model/seir.tsx

```diff
@@ -196,29 +252,42 @@ function getCurveProjections(inputs: SimulationInputs & CurveProjectionInputs) {
     ageGroupFatalityRates.forEach((rate, rowIndex) => {
       const projectionForAgeGroup = simulateOneDay({
         priorSimulation: getAllValues(getRowView(singleDayState, rowIndex)),
-        totalPopulation: sum(ageGroupPopulations),
+        totalPopulation,
         totalInfectious,
-        rateOfSpreadFactor,
+        rateOfSpreadCells,
+        rateOfSpreadDorms,
         pFatalityRate: rate,
+        facilityDormitoryPct,
+        simulateStaff: rowIndex === ageGroupIndex.staff,
       });
       setRowValues(singleDayState, rowIndex, projectionForAgeGroup);
     });
 
     // sum up each column to get the total daily projection for each SEIR bucket
     for (let colIndex = 0; colIndex < singleDayState.shape[1]; colIndex++) {
-      currentDayProjections.push(
-        sum(getAllValues(getColView(singleDayState, colIndex))),
+      const incarceratedValues = getAllValues(
+        getColView(singleDayState, colIndex),
```

Comment:
Does `incarceratedValues` include the SEIR values from the staff group?

Reply:
no, on the line directly below this I remove the staff counts from this array. (it's a little obscure because the array gets mutated in place, but the variable name reflects what the final state will be ... I can change that if you think it's needlessly confusing)

Reply:
Cool, I was 99% sure that I didn't understand the JS but wanted to double check. I had a version of the spreadsheet previously where I forgot to remove the staff from the incarcerated totals, so it's top of mind